hnn_core/viz.py

```diff
@@ -331,6 +331,12 @@ def plot_cells(net, ax=None, show=True):
             marker = markers[cell_type]
             ax.scatter(x, y, z, c=color, marker=marker, label=cell_type)
 
+    if net.lfp:
+        x = [e_dict['pos'][0] for e_dict in net.lfp]
+        y = [e_dict['pos'][1] for e_dict in net.lfp]
+        z = [e_dict['pos'][2] for e_dict in net.lfp]
+        ax.scatter(x, y, z, c='r', s=100, marker='o', label='Electrode')
```

Comment:
this broke something. I get this plot:

![image](https://user-images.githubusercontent.com/15852194/117717104-5e523500-b1a8-11eb-94e7-9f2ff42c8643.png)

Reply:
Yes. Segment locations are given in the NEURON (x,y,z) system, where apical dendrites are along Y. We are plotting (x,z,y). This needs to be fixed in a systematic way.